internal/manager/task_scan.go

```diff
@@ -378,6 +378,11 @@ func walkFilesToScan(s *models.StashConfig, f filepath.WalkFunc) error {
 				return filepath.SkipDir
 			}
 
+			ignore_exists, _ := fsutil.FileExists(filepath.Join(path, ".ignore"))
+			if ignore_exists {
+				return filepath.SkipDir
```

Comment:
Perhaps we should settle on a .stashignore name or something, if this isn't going to be configurable.